src/ContainerInterface.php

```diff
@@ -19,7 +18,7 @@
      * @param string $id Identifier of the entry to look for.
      *
      * @throws NotFoundException  No entry was found for this identifier.
-     * @throws ContainerException Error while retrieving the entry.
+     * @throws \Exception Error while retrieving the entry.
```

Comment:
I think this line confuses people. `@throws \Exception` doesn't make sense indeed. It should've been removed.